skops/temporary_template.md

```diff
@@ -0,0 +1,70 @@
+---
```

Comment:
why is this file here? We should create a temp file on the fly and modify that, and shouldn't have this in the repo.